.rubocop.yml

```diff
@@ -1,5 +1,12 @@
 inherit_from: .rubocop_todo.yml
 
+AllCops:
+  TargetRubyVersion: 1.9
```

Comment:
What does this do? Ideally Rubocop should be run under all the ruby versions tested in Travis.